helm/adelphi/kind-values.yaml

```diff
@@ -0,0 +1,5 @@
+############################ 
+# Config overrides for KIND 
+############################
+
+storageClassName: standard
```

Comment:
Overrides the `local-path` storage class that's the default value in Adelphi (for k3d).

Reply:
This is not used by the tests, it's just a sample config for users running on Kind (similar to what we have for GKE).